[2025-08-25T15:23:07.249+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-08-25T15:23:07.297+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: sales_data_ingestion.extract_sales_data manual__2025-08-25T15:23:03.231373+00:00 [queued]>
[2025-08-25T15:23:07.311+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: sales_data_ingestion.extract_sales_data manual__2025-08-25T15:23:03.231373+00:00 [queued]>
[2025-08-25T15:23:07.311+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-08-25T15:23:07.333+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): extract_sales_data> on 2025-08-25 15:23:03.231373+00:00
[2025-08-25T15:23:07.345+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'sales_data_ingestion', 'extract_sales_data', 'manual__2025-08-25T15:23:03.231373+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/sales_ingestion_dag.py', '--cfg-path', '/tmp/tmpqmufrnz_']
[2025-08-25T15:23:07.349+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=389) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-08-25T15:23:07.350+0000] {standard_task_runner.py:91} INFO - Job 3: Subtask extract_sales_data
[2025-08-25T15:23:07.350+0000] {standard_task_runner.py:63} INFO - Started process 391 to run task
[2025-08-25T15:23:07.414+0000] {task_command.py:426} INFO - Running <TaskInstance: sales_data_ingestion.extract_sales_data manual__2025-08-25T15:23:03.231373+00:00 [running]> on host a755f5a81e3f
[2025-08-25T15:23:07.525+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='sales_data_ingestion' AIRFLOW_CTX_TASK_ID='extract_sales_data' AIRFLOW_CTX_EXECUTION_DATE='2025-08-25T15:23:03.231373+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-08-25T15:23:03.231373+00:00'
[2025-08-25T15:23:07.527+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-08-25T15:23:07.821+0000] {logging_mixin.py:188} INFO - Reading data from /opt/***/data/Orders.csv...
[2025-08-25T15:23:07.914+0000] {logging_mixin.py:188} INFO - Successfully read the data. Here's a preview:
[2025-08-25T15:23:07.960+0000] {logging_mixin.py:188} INFO -    Row ID  ...     Sales
0       1  ...  261.9600
1       2  ...  731.9400
2       3  ...   14.6200
3       4  ...  957.5775
4       5  ...   22.3680

[5 rows x 18 columns]
[2025-08-25T15:23:07.962+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-08-25T15:23:07.963+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-08-25T15:23:07.981+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=sales_data_ingestion, task_id=extract_sales_data, run_id=manual__2025-08-25T15:23:03.231373+00:00, execution_date=20250825T152303, start_date=20250825T152307, end_date=20250825T152307
[2025-08-25T15:23:08.047+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-08-25T15:23:08.066+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-08-25T15:23:08.075+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
